packages/cli/test/integration.test.ts

```diff
@@ -202,13 +270,13 @@ const loginApiServer = require('http')
     console.log(`[mock-login-server] Listening on ${loginApiUrl}`);
   });
 
-const execute = (args, options) =>
+const execute = (args: string[], options?: _execa.Options<string>) =>
   execa(binaryPath, [...defaultArgs, ...args], {
     ...defaultOptions,
     ...options,
   });
 
-const apiFetch = (url, { headers, ...options } = {}) => {
```

Comment:
What type is this supposed to be?

Reply:
I'm guessing its the same params as `fetch()`

Reply:
Updated: [`5acb8bf` (#9672)](https://github.com/vercel/vercel/pull/9672/commits/5acb8bf06d37985ee09d6a001c38f7664e6f1732)